src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -76,30 +74,20 @@ class Trainer:
         returns the test loss, accuracy and runtime
     """
 
-    def __init__(self, experiment_name: str, device: str = None, multi_gpu: Union[MultiGPUMode, str] = MultiGPUMode.OFF,
-                 model_checkpoints_location: str = 'local',
-                 overwrite_local_checkpoint: bool = True, ckpt_name: str = 'ckpt_latest.pth',
-                 post_prediction_callback: DetectionPostPredictionCallback = None, ckpt_root_dir: str = None,
-                 train_loader: DataLoader = None, valid_loader: DataLoader = None, test_loader: DataLoader = None,
-                 classes: List[Any] = None):
+    def __init__(self, experiment_name: str, device: str = None,
+                 multi_gpu: Union[MultiGPUMode, str] = MultiGPUMode.OFF, ckpt_name: str = 'ckpt_latest.pth',
+                 ckpt_root_dir: str = None):
         """
 
         :param experiment_name:                      Used for logging and loading purposes
         :param device:                          If equal to 'cpu' runs on the CPU otherwise on GPU
         :param multi_gpu:                       If True, runs on all available devices
```

Comment:
can't we move multi gpu mode to training params as well?

Reply:
No, its important that in the constructor we specify whether we use DDP or not, as we initialise DDP in the constructor We do so because of the upcoming objects that would be initialised accordingly (for instance the dataloaders will use an appropriate sampler after doing so).

Reply:
ok